### DISCUSSION TITLE
Adding support for e-mail clients

### DISCUSSION THREAD

**Discussion by EgorKuzevanov** [atauenis/webone#50] [Ideas]
Are you planning to add support for old e-mail clients to WebOne? I am interested in supporting the following email clients:

- Outlook Express (version 4.0 and higher)
- Microsoft Outlook (version 97 and higher)
- Mozilla Thunderbird (version 1.0 and higher)
- e-mail clients included in Netscape (version 3.04 and higher)
- e-mail clients included in Mozilla Suite (version 0.9.4 and higher)
- e-mail clients included in SeaMonkey (version 1.0 and higher)
- e-mail clients included in Opera Presto (version 4.02 and higher)

**Comment by atauenis** [comment]
Yes, I am planning to add support for POP3, SMTP in future releases of WebOne. However this needs a large work, and I even don't plan this for this year (if there still will be no additional developers in the project).
